src/Orleans/Serialization/BuiltInTypes.cs

```diff
@@ -1850,6 +1852,7 @@ internal static void SerializeInvokeMethodRequest(object obj, ISerializationCont
         internal static object DeserializeInvokeMethodRequest(Type expected, IDeserializationContext context)
         {
             int iid = context.StreamReader.ReadInt();
+            ushort iVersion = context.StreamReader.ReadUShort();
```

Comment:
If you don't do something like that, then you wont be able to read `InvokeMethodRequest` instances from older silos. I'm not saying it needs to be done, just how it could be done.